web/src/components/Link.tsx

```diff
@@ -0,0 +1,12 @@
+export function Link({ href, linkText }: { href: string; linkText: string }) {
```

Comment:
I will fix the upcoming merge conflict here once https://github.com/electricitymaps/electricitymaps-contrib/pull/6786 is merged :)